### DISCUSSION TITLE
Github

### DISCUSSION THREAD

**Discussion by sanguinafils** [community/community#147057] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Comment envoyé un document dans github

**Comment by queenofcorgis** [comment]
Hi @sanguinafils  , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be closing and locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.

Please note: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
